watcher/local/watcher_test.go

```diff
@@ -141,6 +141,22 @@ func Test_Watcher_Working(t *testing.T) {
 
 			rs.AssertExpectations(t)
 		})
+
+		t.Run("happy/progressed_event", func(t *testing.T) {
+			params, txs := randomTxsForSingleCh(rng, 3)
+			adjSub, trigger := setupAdjudicatorSub(makeProgressedEvent(txs[2])...)
+
+			rs := &mocks.RegisterSubscriber{}
+			rs.On("Subscribe", mock.Anything, mock.Anything).Return(adjSub, nil)
+			w := newWatcher(t, rs)
+			// Start watching and publish states.
+			statesPub, eventsForClient := startWatchingForLedgerChannel(t, w, makeSignedStateWDummySigs(params, txs[0].State))
+			require.NoError(t, statesPub.Publish(txs[1]))
```

Comment:
I think the state publishing is not needed here.

Reply:
Yes, I have removed it. See d20c6b0.